Description: Chainload grubx64.efi, not grub.efi
 We qualify the second stage bootloader image with the architecture name,
 so we're forwards-compatible with any future 32-bit implementations.
 (Non-SB grub doesn't conflict, since the image will be named bootia32.efi
 anyway, not grub.efi.)
Author: Steve Langasek <steve.langasek@ubuntu.com>

--- shim-0~20120906.bcd0a4e8.orig/shim.c
+++ shim-0~20120906.bcd0a4e8/shim.c
@@ -40,7 +40,7 @@
 #include "shim.h"
 #include "signature.h"
 
-#define SECOND_STAGE L"\\grub.efi"
+#define SECOND_STAGE L"\\grubx64.efi"
 
 static EFI_SYSTEM_TABLE *systab;
 static EFI_STATUS (EFIAPI *entry_point) (EFI_HANDLE image_handle, EFI_SYSTEM_TABLE *system_table);
